FirebaseDatabase/Sources/Realtime/FWebSocketConnection.m

```diff
@@ -251,15 +252,15 @@ - (void)send:(NSDictionary *)dictionary {
     // Then, actually send the segments.
     for (NSString *segment in dataSegs) {
 #if TARGET_OS_WATCH
-        [self.webSocketTask sendMessage:[[NSURLSessionWebSocketMessage alloc]
-                                            initWithString:segment]
-                      completionHandler:^(NSError *_Nullable error) {
-                        if (error) {
-                            FFWarn(@"I-RDB083018",
-                                   @"Error sending web socket data: %@", error);
-                            return;
-                        }
-                      }];
+        [self sendStringToWebSocket:segment
+                         completion:^(NSError *_Nullable error) {
+                           if (error) {
+                               FFWarn(@"I-RDB083018",
+                                      @"Error sending web socket data: %@",
+                                      error);
+                               return;
+                           }
+                         }];
 #else
         [self.webSocket send:segment];
```

Comment:
Optional: we probably could have a method like `sendStringToWebSocket:` where we send the string to one of the socket implementations, so the method usage is simpler. But I'm not too strong about it if having different log messages on the error if critical.

Reply:
I do like having the log messages there for logging purposes, especially given that I'm curious to see any failures for watchOS in the wild and what the issues are. Definitely something to consider in the future though, thank you!